includes/polling/applications/docker.inc.php

```diff
@@ -37,33 +22,73 @@ function convertToBytes(string $from): ?int
     return;
 }
 
+$version = intval($version);
+
+if ($version == 1) {
+    $output = 'LEGACY';
```

Comment:
@murrant is this how we usually do this?

Reply:
I took the dhcp stats as a reference: https://github.com/librenms/librenms/blob/master/includes/polling/applications/dhcp-stats.inc.php#L29